_docs/concepts/policy-and-control/mixer.md

```diff
@@ -44,7 +44,7 @@ streams intended for both the service operator as well as for service consumers.
 
 These mechanisms are applied based on a set of [attributes](./attributes.html) that are
 materialized for every request into Mixer. Within Istio, Envoy depends heavily on Mixer. Services running within the mesh
-can also use Mixer to report telemetry or manage quotas. (Note: as of Istio {{ site.data.istio.version }}, only Envoy can call Mixer.)
+can also use Mixer to report telemetry or manage quotas. (Note: as of Istio {{ site.data.istio.version }}, Mixer's API is meant solely for proxy (example: Envoy) use.)
```

Comment:
the Mixer to .., meant for consumption solely by the Istio sidecar.

Reply:
I've updated this to just remove any mention of services calling into Mixer. I think that just muddles the water for the overview.

Reply:
proxy to Istio sidecar, as per our style guide.